nuxt-app/package.json

```diff
@@ -24,7 +24,7 @@
     "@justeattakeaway/pie-cookie-banner": "0.20.5",
```

Comment:
Ideally this should be updated to the latest version to avoid conflicts, because `pie-webc@0.5.17` depends on `pie-cookie-banner@0.23.0`